resources/groups.go

```diff
@@ -27,6 +28,14 @@ func getGroupsClient() resources.GroupsClient {
 	return groupsClient
 }
 
+func getGroupsClientFromAuthFile() resources.GroupsClient {
+	groupsClient := resources.NewGroupsClient(helpers.SubscriptionID())
+	auth, _ := auth.NewAuthorizerFromFile(groupsClient.BaseURI)
```

Comment:
Nice :) . Actually I have a task to add example with FromEnvironment(), should I just add it here ?

Reply:
Ah, yes, why not.
Since this is just an auth example, I did the easiest operation I could think of (create resource group).